### ISSUE TITLE
Error importing valid OpenApi example 

### ISSUE THREAD

**Issue by atrujillofalcon** [Azure/api-management-developer-portal#294]
**Describe the bug**
When trying to import a valid OpenAPI file it fails because it does not correctly recognize the schema of one of the examples. This file has been validated as correct in editor.swagger.io and other tools.

**To Reproduce**
1. Go 'Add new Api'
2. Select OpenAPI type
3. Import attached OpenAPI file
4. Parsing errors are displayed:

- Parsing error(s): Data and type mismatch found. [#/paths/~1news/get/responses/200/content/application~1json/example/data/news/0/id]

- Parsing error(s): The input OpenAPI file is not valid for the OpenAPI specification https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md (schema https://github.com/OAI/OpenAPI-Specification/blob/master/schemas/v3.0/schema.yaml).

**Expected behavior**
Portal must be capable of import an valid OpenApi example.

**Portal environment**
Managed

**Environment**
 - Operating system: Linux
 - Browser: Firefox
 - Version: 70.0.1

**Additional context**

![DeepinScreenshot_Seleccionar área_20191108124706](https://user-images.githubusercontent.com/12805643/68474523-e22f0800-0225-11ea-9b55-9a2418a0c1aa.png)

**Comment by atrujillofalcon** [comment]
[open_api_yml.txt](https://github.com/Azure/api-management-developer-portal/files/3824403/open_api_yml.txt)

**Comment by azaslonov** [comment]
Hi @atrujillofalcon, thanks for reporting the issue. We'll investigate this.

**Comment by VitaliyKurokhtin** [comment]
This is a known problem. Should be fixed soon.

**Comment by azaslonov** [comment]
@atrujillofalcon, please allow 1-2 weeks for the fix to be deployed. Thank you.

**Comment by akhmaladi** [comment]
@azaslonov, im facing the same problem. has this fix deployed?

**Comment by VitaliyKurokhtin** [comment]
Not yet

**Comment by DrReedo** [comment]
I am also facing this problem.  Has it been fixed?  We can't deploy without this fix.

**Comment by atrujillofalcon** [comment]
@azaslonov why close this issue when is not yet deployed?

**Comment by michaeltnguyen** [comment]
This is an issue with the implementation of OpenApiReader.

https://github.com/microsoft/OpenAPI.NET/issues/407
https://github.com/microsoft/OpenAPI.NET/issues/420
https://github.com/microsoft/OpenAPI.NET/issues/429

We've run into this problem as well, and our workaround is to ensure strings and doubles are not parseable as integers.  For example, if you have zipcode: string, it needs to be "12345-6789".  If it's just "12345", it will be parsed as an integer and then will fail to import.  Similar thing with doubles.  It needs to have a number after the decimal (e.g., 11.2).  If not, it will be parsed as an integer and then will fail to import.

Hopefully the issue is fixed soon.

**Comment by mikebudzynski** [comment]
> @azaslonov @VitaliyKurokhtin why close this issue when is not yet deployed?

The issue is closed, because it's not related to the developer portal (this repository). Thank you @michaeltnguyen for clarifying the scenario.

**Comment by DrReedo** [comment]
> This is a known problem. Should be fixed soon.

When will this be fixed?

**Comment by michaeltnguyen** [comment]
@DrReedo this team just works on the portal, so you'll get a more accurate answer from the folks at https://github.com/microsoft/OpenAPI.NET

**Comment by DrReedo** [comment]
> @DrReedo this team just works on the portal, so you'll get a more accurate answer from the folks at https://github.com/microsoft/OpenAPI.NET

@michaeltnguyen thank you, I will have a look over there.

**Comment by calloncampbell** [comment]
I am getting this error from the Developer Portal.

**Comment by petrkmoch** [comment]
Parsing error(s): The Swagger version specified is unknown. 3.0.1

**Comment by chad-rs** [comment]
I just received this when exporting API from one API Management instance, and trying to import into another.
Using the comment from michaeltnguyen on Jan 13 I fixed some example that should have been string but was presented as integer. 
I also noticed that the api that was being exported had spaces in the name, and that was breaking the keys / #ref.
I updated those. 
Also, one of my request body examples was a simple object, but the request body really should have been an array so I enclosed the object in square brackets and everything imported. I hope this helps someone:

![image](https://user-images.githubusercontent.com/63756226/97006986-4e258180-1506-11eb-8a03-42eefa691797.png)
